filebeat/module/nats/log/manifest.yml

```diff
@@ -0,0 +1,9 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - /var/log/nats/nats.log
```

Comment:
Is there just 1 log file or is there also log rotation?

Reply:
No @ruflin, there is no rotation. Actually, `nats` itself by default does not [log into a file](https://nats.io/documentation/server/gnatsd-logging/), and it is user's responsibility to start it with the appropriate flag and path in order to log there. 

Here for example, I as a user, started nats with `./gnatsd -DV -m 8222 -l /var/log/nats/nats.log`.
Make sense? 
Also, do you think we should document it somehow?

Reply:
An addition to @ChrsMark comment: Nats can support log rotate as an external service because it can receive SIGUSR1 and when it does, it reloads the log file. But as Chris mentions, it is up to user to do anything he wants and not implemented inside NATS. [(Paragraph:  Log rotation with logrotate](https://nats.io/documentation/server/gnatsd-logging/))

If we change /var/log/nats/nats.log to /var/log/nats/nats.log* that means that we cover the scenario that user uses log rotate? Furthermore, is it something we want to do?

Reply:
It's interesting to see that services which were mainly built for docker don't even log to files by default. My suggestion here would be that we document on how a user could be logging to a file (with log rotation) or link the docs (see link above). But we should also mention that in most cases, we expect users to just use the docker input with it.

Assuming the rotated logs are `/var/log/nats/nats.log.1`, yes the above would cover it. And yes I think it's the recommended approach.

But as it seems, it's rare that users will actually log directly to a file but normally will leave it to docker to write it from stdout? So I would focus the docs work on that use case.